.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
add the following lines and see if anything changes:

```
--color
--format documentation
```

Reply:
added the line and got the output in a better format bhaiya